### PR TITLE
mypy  linter

### PR DESCRIPTION
**Pull request by babenek**
- linter mypy added
- some of warnings were fixed
- test coverage should be not less 80%
- added unit tests to bring up the coverage
- removed statuses update due it not for inter-fork

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -67,16 +67,50 @@ jobs:
...
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install --requirement requirements.txt
-    - name: UnitTest with pytest
+        pip install --upgrade pip
+        pip install --requirement requirements.txt
+    - name: UnitTest with pytest and coverage
       run: |
-        python -m pytest --random-order --random-order-bucket=global --ignore=docs --ignore=tests/test_app.py tests/
+        python -m pytest \
```
Comment (csh519) [inline .github/workflows/test.yml:74]: It seems you removed `-m` option from python command line.. but it still exist here.
If you want to unify command line style, please remove `-m` option here.

Comment (babenek) [inline .github/workflows/test.yml:74]:
```suggestion
        pytest \
```

**Comment by csh519** [comment]
There are still `-m` option cases.. Please check again.

File: requirements.txt [hunk 2]
```diff
@@ -19,9 +21,13 @@ PyYAML==5.4.1
...
 requests==2.25.1
 scikit-learn==0.24.1
-onnxruntime==1.11.0
 toml==0.10.2
 typing_extensions==4.1.1
-yapf==0.30.0
-nltk==3.6.6
 whatthepatch==1.0.2
+yapf==0.30.0
+pylint
```
Comment (kmnls) [inline requirements.txt:28]: Is there any reason that requirements for tests are the same as requirements for the main feature?

Comment (babenek) [reply requirements.txt:28]: Main feature uses setup.py. 
requirements.txt is necessary for development

**Review by csh519 — APPROVED** [review]
Thank you for adding mypy linter and clean code.

LGTM :+1:

**Review by Dmitriy-NK — APPROVED** [review]
LGTM

**Pull request merged**